### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by prestoncraw**
Added Pagination to the AlarmGroupView Table in `SPCTools` AlarmGroup component to fix issue with controller. I also cleaned up the row and col divs in the `AlarmGroup` component to fix issue with overflow css properties triggering when they shouldn't.  Added a PQBrowser.Url to SystemCenter.Setting table for openSEE.

### CODE REVIEW

**Comment by clackner-gpa** [comment]
@prestoncraw Can you wait until #217 is merged then rebase and change `isPivot` in the SearchFilter` to `IsPivot`? otherwhise searches won't work

**Comment by prestoncraw** [comment]
> @prestoncraw Can you wait until #217 is merged then rebase and change `isPivot` in the SearchFilter`to`IsPivot`? otherwhise searches won't work

Will do

**Review by clackner-gpa — CHANGES_REQUESTED** [review]

**Review by clackner-gpa — APPROVED** [review]

**Pull request merged**
